config/application.yml

```diff
@@ -0,0 +1,2 @@
+ADM_USR: "TheCook"
+ADM_PSW: "hehebwai"
```

Comment:
you should split it be env, for production you should use data from outside the repo